apps/store/src/components/PriceCalculator/CurrentInsuranceField/CurrentInsuranceField.tsx

```diff
@@ -109,13 +109,22 @@ const useUpdateExternalInsurer = (priceIntentId: string) => {
   }
 }
 
-const StyledDialogContent = styled(Dialog.Content)({
+const StyledDialogContent = styled(Dialog.Content)(({ theme }) => ({
+  height: '100%',
   display: 'flex',
-  alignItems: 'center',
-})
+  alignItems: 'flex-end',
+  justifyContent: 'center',
+  paddingTop: theme.space[2],
+  paddingLeft: theme.space[2],
+  paddingRight: theme.space[2],
+}))
 
 const StyledDialogWindow = styled(Dialog.Window)(({ theme }) => ({
-  marginInline: theme.space[2],
-  marginTop: theme.space[2],
-  borderRadius: theme.radius.xs,
+  width: '100%',
+  maxWidth: 700,
```

Comment:
Should we convert these to relative units?

```suggestion
  maxWidth: '43.75rem',
  ...
  maxHeight: '49.38rem',
```

Btw, the following can be a nit trick:

```
maxWidth: 'calc(700rem / 16)'; /* ~700px */
```

It's easier to think about px but in general we might want relative sizes for dimensions (width, height, etc) so this could be a way to "write it in px" but actually use rems, with the tradeoff of giving some calculation to the browser. I've mixed feelings about it. What do you think?

Reply:
Ahh I agree with you.

However, these values are decided by Insurely - I should've commented that better. We don't control them - it's just the max values of the iframe that Insurely has decided.